source/payment-service-provider.html.erb

```diff
@@ -83,6 +83,16 @@ title: GOV.UK Pay’s Payment Service Provider
         <p class="govuk-body">There are no overhead costs on the account. You don’t pay a management, set up or monthly fee. You just pay the transaction charges. Nothing more.</p>
         <p class="govuk-body">Transaction fees are commercially sensitive. Details are available in the contract when you sign in to your account.</p>
 
+        <h2 class="govuk-heading-m">Setting up a Stripe test account</h2>
+        <p class="govuk-body">You can test Stripe’s reporting before your service goes live. You can access the same reports and functionality as a live account, including:</p>
+        <ul class="govuk-list govuk-list--bullet">
+          <li>transaction fees</li>
+          <li>gross and net payments</li>
+          <li>payments into your bank account</li>
+        </ul>
+        <p class="govuk-body">Download the data as a .csv file or by integrating with our reporting and reconciliation API.</p>
+        <p class="govuk-body">Request a Stripe test account from your service's dashboard in the Admin tool.</p>
```

Comment:
Shouldn't we include a sort of link to Dashboard?

Reply:
We can't have hard links anymore as all URLs are based on account external ID. Section `Request Stripe test account` should be visible on dashboard and tech docs will cover some instructions.

Reply:
👀  we can still link to https://selfservice.payments.service.gov.uk/dashboard which will go through to the dashboard page without an issue. If that's functionality we'd like to keep around we can plan it into the URL structure cleanup.

In this case it would just go the last selected account so maybe it's best for the user to explicitly choose the right account before seeing the request a test account page (for example it won't be an option if the last account they had selected was a live account).